elixir/cowboy/config.yaml

```diff
@@ -1,3 +1,4 @@
 framework:
   website: https://ninenines.eu/docs/en/cowboy/2.7/guide/
```

Comment:
no need to have `website:` **and** `github` -> if there is a `website`, prefer it
+
no need to use `https://` prefix for **website** (it is added automatically)